docs/content/specs/terraform/_index.md

```diff
@@ -533,6 +543,466 @@ Bad examples:
 var.new_network_security_group_name == null ? "${var.subnet_name}-nsg" : var.new_network_security_group_name)
 ```
 
+<br>
+
+---
+
+<br>
+
+#### ID: TFFR14 - Category: Inputs - No `enabled` or `module_depends_on` variable
+
+Since Terraform 0.13, `count`, `for_each` and `depends_on` are introduced for modules, module development is significantly simplified. Module's owners **MUST NOT** add variables like `enabled` or `module_depends_on`).
```

Comment:
I also am not clear what we mean here - can we provide examples?

Reply:
```suggestion
Since Terraform 0.13, `count`, `for_each` and `depends_on` are introduced for modules, module development is significantly simplified. Module's owners **MUST NOT** add variables like `enabled` or `module_depends_on` to control the entire module's operation. Boolean feature toggles are acceptable however.
```